src/storage/storage.ts

```diff
@@ -72,6 +72,21 @@ export class Storage implements IStorage {
     this.version = version;
   }
 
+  /**
+   * Return the deposit percentage for creating tokens.
+   * @returns {number}
+   */
+  getTokenDepositPercentage(): number {
+    if (this.version && this.version.token_deposit_percentage) {
+      return this.version.token_deposit_percentage;
+    }
+    /**
+     *  When using wallet-service facade we do not update the version constants
+     *  Since this data is important for the wallets UI we will return the default value here.
+     */
+    return TOKEN_DEPOSIT_PERCENTAGE;
```

Comment:
The wallet service has a table for constants, and it also needs this percentage information. How was it handled before?

Reply:
Good observation, after checking I saw that the `getDepositPercentage` from the token utils would retrieve the deposit from storage (it was set by the version api, which the wallet-service does not use, hence the default) and this would be used internally by other token utils, this is why we had the default in the method instead of the wallet-service.

I'm changing so that the value can be received via argument, and use the default if not present, all instances on the wallet should retrieve from storage and pass as an argument and the other token utils also receive the storage as a parameter so they can get it from there as well, sounds good?